docs/tutorials/intro_tutorial.rst

```diff
@@ -990,7 +1006,35 @@ iteration).
 
     245it [00:25,  9.75it/s]
 
+**Note for Windows OS users:** You may have an issue with `batch_run` and `number_processes = None`. Your cell will
```

Comment:
I think we need a stronger statement, "you will" instead of "you may"? Isn't it happening 100% of the time?